DataMapping/EntityDataMapperService.php

```diff
@@ -33,7 +33,7 @@ class EntityDataMapperService
     /**
      * @since 1.1.0
      *
-     * @var RegistryInterface
+     * @var ManagerRegistry

```

Comment:
@cravler , please check and confirm that both Doctrine v1 and v2 have Doctrine\Common\Persistence\ManagerRegistry interface available.

Reply:
yes, they have

doctrine/doctrine-bundle v1.0 -> doctrine/dbal v2.2 -> doctrine/common v2.2
https://github.com/doctrine/common/blob/2.2.0/lib/Doctrine/Common/Persistence/ManagerRegistry.php

doctrine/common v2.9 -> doctrine/persistence v1.0
https://github.com/doctrine/persistence/blob/1.0.x/lib/Doctrine/Common/Persistence/ManagerRegistry.php